### DISCUSSION TITLE
sub region not working

### DISCUSSION THREAD

**Discussion by Endlessvoid303** [maruohon/litematica#647] [General]
how do i make a sub region when i click the button nothing hapens

**Comment by maruohon** [comment]
There are two main "area selection modes" in the mod: `Simple` and `Normal` (which will be renamed to `Multi-Region` to be more descriptive). You can switch between those modes in the Litematica main menu, in the middle left.

 The `Simple` mode only ever has one sub-region, you can't add more and you can't delete that one box. That mode is meant for simple "temporary"/quick one-box selections.

The `Normal`/`Multi-Region` mode on the other hand is meant for, well, multi-region selections, and this mode also stores each selection to a separate file on disk, so that you can keep as many separate selections around as you want, and you go back to them later if needed, without having to re-do possibly very complex multi-box selections. Fox example let's say you are designing a new world eater in a creative world, and you create a multi-box selection for it, and save it. But then you later notice that something needs fixing, and you fix it in this same "master copy" of the design. You can then just open that earlier selection again and save a new version of it, without having to re-do the selection, even if you worked on other stuff in the same world since the previous time you saved this contraption.

So to work with sub-regions, you first need to be in the `Normal` mode. And then you create or open an existing selection save file via the `Area Selection Browser` menu. After that you use the `Area Editor` menu to create and rename the sub-region boxes. You can also use it to initially select the sub-regions to work with/move using the tool (by clicking on one so that it has the white outline). This is useful if you create multiple boxes at once and they are all overlapped in a single spot in the world, so selecting a particular box by middle clicking with the tool wouldn't be feasible yet before you spread them out first.
